Java.gitignore

```diff
@@ -22,3 +22,12 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 replay_pid*
+
+# Gradle specific files
+.gradle/
+build/
+
+# Eclipse specific files
+.settings/
+.project
+.classpath
```

Comment:
We have an [eclipse specific](https://github.com/github/gitignore/blob/main/Global/Eclipse.gitignore) template and would prefer to not include IDE specific config in language templates.